### PR TITLE
Flyway blog post

### PR DESCRIPTION
**Pull request by brunokrebs**

### CODE REVIEW

File: _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown [hunk 1]
```diff
@@ -0,0 +1,431 @@
+---
```
Comment (brunokrebs) [inline _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown]: Watch out, many sentences here start with "But" and "So". I have been warned by some that this is not correct but I have researched and there are many resources out there that claim that it is indeed not a problem.

But :), feel free to criticize

File: _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+}
+```
+
+And then configure three parameters in the Auth0 SDK, which is done by adding the following properties in the `application.properties` file:
+
+```bash
+# change this with your own Auth0 client id
+auth0.audience=LtiwyfY1Y2ANJerCNTIbT7vVsX5zKBS5
+# change this with your own Auth0 client secret
+auth0.secret=TjpxsT2pMt9Jj6Np45GSPnTnHY-Y-LFyv6fUGGH_EGQLD4_ONBuymn3zxfcCnpdJ
```
Comment (sebadoom) [inline _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown]: Is this secret meant to be public in the post?

File: _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+
+![Retrieving contacts of a customer](https://cdn.auth0.com/blog/spring-boot-flyway/get-contacts.png)
+
+## Conclusion
+
+Having a tool like Flyway integrated in our application is a great addition. With it, we can create scripts that will refactor the database to a state that is compatible with the source code, and we can move data around to guarantee that it will reside in the correct tables.
+
+Flyway will also be helpful if we eventually run into an issue where we need to recover the database from a backup. In a case like that, we can rest assured that Flyway will correctly identify if the combination of the source code version and the database version that we are running have any scripts that need to be applied.
+
+## Aside: Securing Java Applications with Auth0
```
Comment (sebadoom) [inline _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown]: AFAIK, the aside should come before the conclusion. Have you checked with Tony about this?

File: _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown (hunk 2 above)
Comment (brunokrebs) [inline _posts/2017-03-29-incrementally-changing-your-database-with-java-and-flyway.markdown]: I keep it here as an example, it is not a valid secret actually. I have changed a few chars on it.

**Pull request merged**